container_pipeline/lib/command.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+import subprocess
+
+
+def run(cmd):
+    """
+    Runs a shell command.
+
+    :param cmd: Command to run
+    :type cmd: str
+
+    :return: Command output
+    :rtype: str
+    :raises: subprocess.CalledProcessError
+    """
+    return subprocess.check_output(cmd, shell=True)
```

Comment:
@rtnpro As discussed, please make this function accept `**kwargs` so that if someone's trying to use `shell=True` in any worker, we can ensure its absolute necessity and then merge it.